### DISCUSSION TITLE
Cross-Lib and your project may benefit from a collaboration

### DISCUSSION THREAD

**Discussion by Fabrizio-Caruso** [sehugg/8bitworkshop#192] [Q&A]
Hi!
I am the author of https://github.com/Fabrizio-Caruso/CROSS-LIB
which is a full WORA framework to code for nearly *all* (about 200) vintage computers by using the *very same* source code and even the *very same* assets. You code once and you get the same game on all 8-bit and 16-bit vintage targets (and even on the native PC).

I have written 9 universal games with Cross-Lib. All of them can be built automatically from the very same code and work on about 200 consoles, computers, pocket calculators, arcade boards, hand-held consoles, etc.

Cross-Lib provides:
- a C library that provides a common abstraction layer for all 200 targets
- Python scripts for the build-chain and other tools

Cross-Lib is also available throw a docker image and a devcontainer.

I was wondering whether we could start a collaboration on some topics.

Anyway feel free to use my source code for non-commercial purposes (just mention me as the author if you use it).

**Comment by sehugg** [comment]
Hi Fabrizio, I have seen your cross-lib project before, looks interesting. I'm not sure how much time I have but let me know what you are thinking. (you can email at info at 8bitworkshop dot com)
